jdaviz/configs/imviz/tests/test_linking.py

```diff
@@ -280,64 +282,78 @@ def test_wcslink_offset(self):
-        assert not self.viewer.label_mouseover.unreliable_pixel
+        label_mouseover._viewer_mouse_event(self.viewer,
+                                            {'event': 'mousemove', 'domain': {'x': 10, 'y': 3}})
+        assert label_mouseover.as_text() == ('Pixel x=10.0 y=03.0',
+                                             'World 00h14m19.6291s -30d23m30.9692s (ICRS)',
+                                             '3.5817877198 -30.3919358920 (deg)')
+        assert not label_mouseover.row1_unreliable
+        assert label_mouseover.row2_unreliable
+        assert label_mouseover.row3_unreliable
 
     def test_pixel_linking(self):
         self.imviz.link_data(link_type='pixels')
 
         # Check the coordinates display: Last loaded is on top.
-        self.viewer.on_mouse_or_key_event({'event': 'mousemove', 'domain': {'x': -1, 'y': -1}})
-        assert self.viewer.label_mouseover.pixel == 'x=-1.0 y=-1.0'
-        assert self.viewer.label_mouseover.value == ''
-        assert self.viewer.label_mouseover.world_ra_deg == '3.5816611274'
-        assert self.viewer.label_mouseover.world_dec_deg == '-30.3919634282'
-        assert self.viewer.label_mouseover.unreliable_world
-        assert not self.viewer.label_mouseover.unreliable_pixel
+        label_mouseover = self.imviz.app.session.application._tools['g-coords-info']
+        label_mouseover._viewer_mouse_event(self.viewer,
+                                            {'event': 'mousemove', 'domain': {'x': -1, 'y': -1}})
+        assert label_mouseover.as_text() == ('Pixel x=-1.0 y=-1.0',
+                                             'World 00h14m19.5987s -30d23m31.0683s (ICRS)',
+                                             '3.5816611274 -30.3919634282 (deg)')
+        assert not label_mouseover.row1_unreliable
+        assert label_mouseover.row2_unreliable
+        assert label_mouseover.row3_unreliable
 
         # Back to reference image with bounds check.
-        self.viewer.on_mouse_or_key_event({'event': 'keydown', 'key': 'b',
-                                           'domain': {'x': -1, 'y': -1}})
-        assert self.viewer.label_mouseover.pixel == 'x=-1.0 y=-1.0'
-        assert self.viewer.label_mouseover.value == ''
-        assert self.viewer.label_mouseover.world_ra_deg == '3.5815955408'
-        assert self.viewer.label_mouseover.world_dec_deg == '-30.3919405616'
-        assert self.viewer.label_mouseover.unreliable_world
-        assert not self.viewer.label_mouseover.unreliable_pixel
+        label_mouseover._viewer_mouse_event(self.viewer,
+                                            {'event': 'keydown', 'key': 'b',
+                                             'domain': {'x': -1, 'y': -1}})
+        assert label_mouseover.as_text() == ('Pixel x=-1.0 y=-1.0',
+                                             'World 00h14m19.5987s -30d23m31.0683s (ICRS)',
+                                             '3.5816611274 -30.3919634282 (deg)')

```

Comment:
I think the new sky coordinates in this diff are wrong. The old values in the diff are:
```python
        assert self.viewer.label_mouseover.world_ra_deg == '3.5815955408'
        assert self.viewer.label_mouseover.world_dec_deg == '-30.3919405616'
```

Reply:
fixed (I accidentally updated the test to send the blink command to the wrong listener and just didn't notice the subtle change in coordinates when I pasted the output)